pkg/basic/server.go

```diff
@@ -152,7 +182,7 @@ func loadInlineTrustVector(inline *InlineTrustVector) (TrustVector, error) {
 			return nil, errors.Errorf("entry %d: v=%f is out of range (0, inf)",
 				idx, entry.V)
 		}
-		lt.SetVec(entry.I, entry.V)
+		entries = append(entries, sparse.Entry{Index: entry.I, Value: entry.V})
 	}
-	return lt, nil
+	return sparse.NewVector(inline.Size, entries), nil
```

Comment:
At this point in the code, we would have 3 copies of the TrustVector - 1 copy in the InlineTrustVector, 2nd copy in entries slice, and  3rd copy in sparse.NewVector.

Reply:
Fixed.  Now we have only two copies at most at any given point in time, and only one copy after this function returns:

- `entries`—discarded at the end of this function
- the constructed `*sparse.Vector`—this is the one that lives on.